2200037/5431.py

```diff
@@ -0,0 +1,11 @@
+#민석이의 과제 체크하기
+test = int(input())
+for T in range(1,test+1):
+    n,task_n = map(int,input().split())
+    ans_list = list(map(int,input().split()))
+    n_list = [i for i in range(1,n+1)]
```

Comment:
```suggestion
    n_list = list(range(1,n+1))
```
위와 같이 작성하는 방법도 있습니다 😄